handbook/ce/ce.md

```diff
@@ -0,0 +1,41 @@
+# Customer Engineers
+
+## Our ethos
+
+Customer Engineers at Sourcegraph are a combination of pre-sales sales engineering and post-sales customer success. We own the relationship with our developer users and champions at our customers and prospects. We serve as the technical experts during the sales cycle and help enable our customers throughout our journey together.
```

Comment:
I wonder if there is a different way to say "we own the relationship with our...." to convey the directly responsible individual (DRI) aspect, as well as the tone that our customers are always "our customers" and never "my customer..."

Reply:
Thank you for this. 

How about from: We own the relationship with our developer users and champions at our customers and prospects.

To: We are first and foremost responsible for building and fostering relationships with developer users and champions at our prospects and customers.

Reply:
I love that change!